extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js

```diff
@@ -74,11 +63,85 @@ describe.configure().ifNewChrome().run('amp-lightbox-gallery', function() {
+        expect(controlsContainerQuery.length).to.equal(1);
+        const controlsContainer = controlsContainerQuery[0];
+        expect(controlsContainer.classList
+            .contains('i-amphtml-lbg-single')).to.be.true;
+
+        const closeButton = getButton(win.document,
+            'i-amphtml-lbg-button-close');
+        expect(closeButton.getAttribute('aria-label')).to.equal('Close');
+        expect(win.getComputedStyle(closeButton).display).to.equal('block');
+
+        const galleryButton = getButton(win.document,
+            'i-amphtml-lbg-button-gallery');
+        expect(galleryButton.getAttribute('aria-label')).to.equal('Gallery');
+        expect(win.getComputedStyle(galleryButton).display).to.equal('none');
+
+        const prevButton = getButton(win.document, 'i-amphtml-lbg-button-prev');
+        expect(prevButton.getAttribute('aria-label')).to.equal('Prev');
+        expect(win.getComputedStyle(prevButton).display).to.equal('none');
+
+        const nextButton = getButton(win.document, 'i-amphtml-lbg-button-next');
+        expect(nextButton.getAttribute('aria-label')).to.equal('Next');
+        expect(win.getComputedStyle(nextButton).display).to.equal('none');
+        done();
+      });
+    });
+
+    it('should display text description', done => {
+      openLightbox(win.document).then(() => {
+        const descBoxQuery =
+          win.document.getElementsByClassName('i-amphtml-lbg-desc-box');
+        expect(descBoxQuery.length).to.equal(1);
+        const descBox = descBoxQuery[0];
+        expect(descBox.classList.contains('i-amphtml-lbg-standard')).to.be.true;
+        expect(descBox.children.length).to.equal(2);
+        const descriptionMask = descBox.children[0];
+        expect(descriptionMask.classList.contains('i-amphtml-lbg-desc-mask'))
+            .to.be.true;
+
+        const descriptionText = descBox.children[1];
+        expect(descriptionText.classList.contains('i-amphtml-lbg-desc-text'))
+            .to.be.true;
+        expect(descriptionText.textContent).to.equal('This is a figcaption.');
+        done();
       });
     });
   });
 });
 
+function openLightbox(document) {
+  const lightbox = document.getElementById('amp-lightbox-gallery');
+  expect(lightbox.style.display).to.equal('none');
+  const ampImage = document.getElementById('img0');
+  const imageLoadedPromise = waitForImageToLoad(ampImage);
+  return imageLoadedPromise.then(() => {
+    const ampImage = document.getElementById('img0');
+    // Simulate a click on the img inside the amp-img, because this is
+    // what people tend to actually click on.
+    const openerImage = ampImage.querySelector('img[amp-img-id="img0"]');
+    const openedPromise = waitForLightboxOpen(lightbox);
+    openerImage.click();
+    return openedPromise;
+  });
+}
+
+function getButton(document, className) {
+  const buttonQuery = document.getElementsByClassName(className);
+  expect(buttonQuery.length).to.equal(1);
```

Comment:
nit: `expect`s in getter helpers seems odd to me. It'd be OK to repeat the `expect` in each test